app/permissions.go

```diff
@@ -14,6 +14,7 @@ import (
 )
 
 const permissionsExportBatchSize = 100
+const systemSchemeName = "9CE2A3E8-D8EA-4621-BCC0-27DED244D37B" // Prevents collisions with user-created schemes.
```

Comment:
If we going to include a "Magic number" to define the systemSchemeName, I would prefer something like `00000000-0000-0000-0000-000000000000`, will avoid the collisions and looks like something fixed.